webapp/src/client/client.js

```diff
@@ -1,20 +1,21 @@
-import {Client4} from 'mattermost-redux/client';
-import {ClientError} from 'mattermost-redux/client/client4';
-import AuthenticationContext from 'adal-angular';
+import {Client4} from "mattermost-redux/client";
+import {ClientError} from "mattermost-redux/client/client4";
+import AuthenticationContext from "adal-angular";
 
-import {isDesktopApp} from '../utils/user_utils';
-import {Periods} from '../constants';
-import {id as pluginID} from '../manifest';
+import {isDesktopApp} from "../utils/user_utils";
+import {Periods} from "../constants";
+import {id as pluginID} from "../manifest";
+import {getPluginServerRoute} from "selectors";
```

Comment:
Single quotes are the de-fact standard for js code at Mattermost. Why does this PR change them?

Reply:
@hanzei  sorry about this. This was one of the first PRs that got sent out and this has been ignored for a while. I'll pick this up and clean it up

Reply:
@hanzei  removed this unwanted changes